bravenewcoin-vwap/adapter.js

```diff
@@ -0,0 +1,95 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+
+const customParams = {
+  symbol: ['base', 'from', 'coin', 'symbol', 'assetId', 'indexId', 'asset'],
+  indexType: false,
+  timestamp: false
+}
+
+const host = 'bravenewcoin.p.rapidapi.com'
+
+const authenticate = async () => {
+  const response = await Requester.request({
+    method: 'POST',
+    url: `https://${host}/oauth/token`,
+    headers: {
+      'content-type': 'application/json',
+      'x-rapidapi-host': host,
+      'x-rapidapi-key': process.env.API_KEY,
+      accept: 'application/json',
+      useQueryString: true
+    },
+    data: {
+      audience: 'https://api.bravenewcoin.com',
+      client_id: process.env.CLIENT_ID,
+      grant_type: 'client_credentials'
+    }
+  })
+  return response.data.access_token
+}
+
+const getAssetId = async (token, symbol) => {
+  const response = await Requester.request({
+    url: `https://${host}/asset`,
+    headers: {
+      'content-type': 'application/octet-stream',
+      'x-rapidapi-host': host,
+      'x-rapidapi-key': process.env.API_KEY,
+      useQueryString: true
+    },
```

Comment:
We can also reuse the same API headers here:
```suggestion
    headers: {
      'content-type': 'application/octet-stream',
      useQueryString: true,
      ...apiHeaders
    },
```